poolmgr/api.go

```diff
@@ -178,6 +179,21 @@ func (poolMgr *Poolmgr) getFunctionEnv(m *api.ObjectMeta) (*tpr.Environment, err
 }
 
 func (poolMgr *Poolmgr) getServiceForFunction(m *api.ObjectMeta) (string, error) {
+	// Pool manager uses function metadata to generate a key
```

Comment:
This changes the meaning of the poolmgr API. This bug needs to be solved in the router, I think: it should not be resolving trigger function references to outdated functions.